conf/evolutions/default/3.sql

```diff
@@ -0,0 +1,16 @@
+# --- Indexing on queue for seach by queue feature
+# --- !Ups
+
+alter table yarn_app_result add column resource_used    BIGINT        UNSIGNED DEFAULT 0    COMMENT 'The resources used by the job in MB Seconds';
```

Comment:
Hey paul, I'd already merged the changes since individual PRs have been already reviewed. I'll take your comments into account in the next set of PRs.